docs/index.md

```diff
@@ -7,54 +7,54 @@ computational narratives using the Jupyter ecosystem.
 
 We are the community behind [Jupyter Book](https://jupyterbook.org)--an open source project for building beautiful, publication-quality books and documents from computational material--as well as [MyST Markdown](https://myst-parser.readthedocs.io/en/latest/)--a rich and extensible flavor of Markdown meant for technical documentation and publishing.
 
-::::{grid} 1 2 2 2
+::::\{grid} 1 2 2 2
```

Comment:
I updated the PR to add those plugins to our hook.

Hmm - looks like it is not picking up the colon fence syntax, is there a way to tell the pre-commit hook to re-use the same plugins that are enabled in `conf.py`?

Reply:
https://github.com/executablebooks/mdformat-myst/issues/13 and https://github.com/executablebooks/mdformat-myst/pull/9

But no, not presently 😢

Reply:
ah bummer - ok I'll just comment out the lines about mdformat and we can re-enable if/when there is support for extra myst extensions.